honeybee_energy/load/lighting.py

```diff
@@ -0,0 +1,311 @@
+# coding=utf-8
+"""Complete definition of lighting in an energy simulation, including schedule and load."""
+from __future__ import division
+
+from ._base import _LoadBase
+from ..schedule.ruleset import ScheduleRuleset
+from ..schedule.fixedinterval import ScheduleFixedInterval
+from ..reader import parse_idf_string
+from ..writer import generate_idf_string
+
+from honeybee._lockable import lockable
+from honeybee.typing import float_in_range, float_positive, tuple_with_length
+
+
+@lockable
+class Lighting(_LoadBase):
```

Comment:
would subclassing from `_EquipmentBase` save you writing some of the code here?

Reply:
I thought about this but there's really only two properties that Equipment and Lighting have perfectly in common (`watts_per_area` and `schedule`).  Everything else would have to be different and I think I save myself from the most lines of similar code by making `_EquipmentBase` really good at addressing the needs of both `Equipment` objects rather than trying to get it to work for lighting and both equipment objects.